src/main/java/com/github/fishio/achievements/Achieve.java

```diff
@@ -0,0 +1,75 @@
+package com.github.fishio.achievements;
+
+import com.github.fishio.logging.Log;
+import com.github.fishio.logging.LogLevel;
+
+/**
+ * This class loads a set of Achievements and sets the rules that have to be met
+ * in order for the achievement to be obtained.
+ *
+ */
+public class Achieve {
```

Comment:
*Auto-generated comment*

    Utility classes should not have a public or default constructor.